CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Unpin OTel SDK/API version
     ([#310](https://github.com/microsoft/ApplicationInsights-Python/pull/310))
+- Replace custom log processor exporter interval env var with OT SDK env var
```

Comment:
It's always been the OT env var. Just a question of whether we use it on our end or leave it for the OTel sdk (which previously failed to do so).

Reply:
Changed to read "explicit" instead of custom. No logic is needed in our distro when it pertains to configuring export interval as we are leveraging OT SDK env vars.